kms/quickstart.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample quickstart is a basic program that uses Cloud KMS.
+package kms
+
+// [START kms_quickstart]
+import (
+	"context"
+	"fmt"
+	"log"
+
+	cloudkms "cloud.google.com/go/kms/apiv1"
+	"google.golang.org/api/iterator"
+	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
+)
+
+func main() {
+	projectID := "your-project-id"
```

Comment:
Use an env variable for the project?

```suggestion
	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
	if projectID == "" {
		log.Fatal("Must set the GOOGLE_CLOUD_PROJECT environment variable."
	}
```

Reply:
Using a string like this seems to be a standard used by many quickstarts. I think we should keep it as is to show the minimum viable code, and to be consistent with the other languages